pages/tests/run.py

```diff
@@ -2,9 +2,9 @@
 """From http://stackoverflow.com/a/12260597/400691"""
 import sys
 
+from colour_runner.django_runner import ColourRunnerMixin
```

Comment:
Since [0.7.0](https://github.com/PyCQA/flake8-import-order/commit/ca6b2686ac712c34c9ec863b7785bb4af1370409) `flake8-import-order` has replaced the `google` rules with `smarkets`. This is why we had this import order error.

To fix it you can add `smarkets` to the [`import-order-style` in `setup.cfg`](https://github.com/incuna/feincms-pages-api/blob/501f2d4be3c1c5c00bc4dc4fdd82cf57a418c271/setup.cfg#L9).